### ISSUE TITLE
Can we have a new release?

### ISSUE THREAD

**Issue by mdsantosdev** [gin-gonic/gin#1732]
I'm using Glide as a dependency manager and I'm using a new feature (ShouldBindUri which I guess it has 30 days merged in Master), can you guys create a new version so we can use those new features?

**Comment by Lord-Y** [comment]
Can't wait to have validator v9 in the new release :)

**Comment by mikestead** [comment]
@thinkerou appreciate the work you're doing. I think this issue should perhaps stay open until you cut that release. I came here for same reason and had to go digging to find it.

**Comment by dangkaka** [comment]
hello, any update?

**Comment by mikestead** [comment]
The milestone is due in a day or so https://github.com/gin-gonic/gin/milestone/9 fingers crossed!

**Comment by thinkerou** [comment]
@mikestead thanks! Due date is only an estimate. we still need to wait @javierprovecho come back.

**Comment by visamz** [comment]
Hello, any update?

**Comment by cugxuan** [comment]
Hello, any update?

**Comment by koderoff** [comment]
Any update on this?

**Comment by dmarkham** [comment]
@javierprovecho @thinkerou @appleboy I am willing to help push this over the finish line if that is helpful.

**Comment by A-limon** [comment]
We all looking forward to this release.

**Comment by dmarkham** [comment]
@thinkerou  the `vendor/vendor.json` versions I don't think are matching up with the `go.mod` / `go.sum`  files what is your thoughts for dealing with this? Are we dropping support for what ever was using `vendor/vendor.json` this release?

**Comment by thinkerou** [comment]
@dmarkham need to update `vendor/vendor.json`. gin1.4 need to support govendor because go module releases on go1.11.

**Comment by dmarkham** [comment]
@thinkerou  so after taking a stab at getting the `govender` to match `go modules` I'm not yet able to to get `go1.6` to work.  The main reason is we have moved to version`v0.04` ->  `v0.0.6` of `github.com/mattn/go-isatty` This package is now using the  `golang.org/x/sys/unix` pacakage  that then t uses `runtime.KeepAlive` this is *not* in `go1.6`.  So This leaves us a few options. 

1. downgrade `github.com/mattn/go-isatty`
2. drop support for `go1.6` and `go1.7` in  gin `1.4.0`
3. use different versions of `github.com/mattn/go-isatty` depending on your choice of dependency manager/go version. 

My preference is to drop `go1.6` and `go1.7` in `1.4.0`.

Thoughts?

**Comment by Lord-Y** [comment]
Drop it cloud be a good choice as we're already in go version 1.12.

**Comment by dmarkham** [comment]
@thinkerou @appleboy  do you guys have any feedback that we can use to move forward on this? or we planning just to hold off and wait for @javierprovecho?

**Comment by dmarkham** [comment]
Went ahead and merged in the removal of `go1.6`/`go1.7`  #1851  in a new pull request as an option for @javierprovecho.

**Comment by rssathe** [comment]
@thinkerou @appleboy  dont think that such a widely used repo should be blocked on merging improvements if @javierprovecho is away

**Comment by appleboy** [comment]
@thinkerou and me don't have the permission to create new Tag and we must wait for @javierprovecho review all features and bugs.

**Comment by visamz** [comment]
so, can you contact with the author?

**Comment by dmarkham** [comment]
Yes @visamz  Javier has been in contact.  He is currently traveling. I do expect things to speed up a bit when he is back. I selfishly would like to have `gin` a bit more nimble, but it is such a delicate balance between speed and stability. 
I have done just a small amount of the grunt work for this release, but I have not setup for a gin release before, so all of the work needs to be checked/verified/corrected and may turn out not be helpful at all!  

We already have so many new PR's ready for `v1.5.0` killing me to have people waiting to get code merged in or explained why it will be rejected.

`v1.4.0` should be here soon enough :100:

**Comment by vankofalcon** [comment]
:wave: Guys, do we have an approximate date when master will be released?

**Comment by awkj** [comment]
Waiting

**Comment by anasanzari** [comment]
still waiting

**Comment by LuGuoHuas** [comment]
emm... few months later.

**Comment by dmarkham** [comment]
:disappointed:

**Comment by goldyfruit** [comment]
`go.mod` feature allows you to specify a specific version, example:
```
module mymodule
go 1.12
require (
	github.com/gin-gonic/gin master
)
```
Go 1.11 is required.

**Comment by dmarkham** [comment]
@appleboy  @thinkerou  @javierprovecho  Hello Guys! Just a friendly reminder, we are all excited for a new release.

**Comment by appleboy** [comment]
See the new release v1.4.0 https://github.com/gin-gonic/gin/releases/tag/v1.4.0

**Comment by dmarkham** [comment]
@thinkerou  @appleboy  Nice job guys I know it was not easy... Lets get rolling on `1.5.0` :+1:
